docs/resources/ngwaf_datadog_alert.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: "fastly"
+page_title: "Fastly: ngwaf_datadog_alert"
+sidebar_current: "docs-fastly-resource-ngwaf-datadog-alert"
+description: |-
+  Provides a Fastly Next-Gen WAF Redaction for a Workspace
+---
+
+# ngwaf_datadog_alert
+
+Provides a Fastly Next-Gen WAF Datadog Alert, which provides a connection to Datadog.
+
+## Example Usage
+
+Basic usage:
+
+```terraform
+resource "fastly_ngwaf_datadog_alert" "demo_datadog_alert" {
+  description                  = "Some Description"
+  integration_key              = "123456789"
+  integration_site             = "us1"
+  workspace_id                 = fastly_ngwaf_workspace.demo.id
+}
+```
+
+## Import
+
+Fastly Next-Gen WAF Datadog Alerts can be imported using their ID and the ID of the workspace they belong to.
+
+```sh
+$ terraform import fastly_ngwaf_datadog_alert.example <workspace_id>/<alert_id>
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `integration_key` (String) The Datadog integration key.
+- `integration_site` (String) The Datadog site.
```

Comment:
Is there a reason these have the 'integration_' prefix? In the new API model these item are referred to as Alerts so we should probably avoid using 'integration' except when the API requires it, but in this case I don't think either of these need a prefix at all: "key" and "site" should be fine?